devdocs/process.md

```diff
@@ -197,31 +197,24 @@ Role: PM
 
 
 ###Applying a fix
-Role: committer
-
-  * Do not merge online. Always merge locally and push to the repo. If you 
-    merge online, you will not be able to run tests or use the required format
-    for the commit message.
-  * Fetch code from upstream: <br>
-    `git fetch origin`<br>
-  * Checkout the branch and update with latest master<br>
-    `git checkout -b 2287-add-sample-course-test origin/2287-add-sample-course-test`<br>
-    `git merge master` <br>
-  * Test the code by running the `Local tests` and ensure that the all tests pass on Travis.
-  * If green, 
-    * Merge to master and push.<br>
-      `git checkout master` <br>
-       Merge the branch. Format of the commit message: 
-       `[Issue number] Issue title as given in the original issue`<br>
-       e.g. `[2287] Add more tests for newly joined Instructor accessing sample course`
-    * Remove any status labels from the pull request. Delete the branch (from GitHub UI).
-    * Remove any status labels from the corresponding issue and close it.
-    * Optionally, apply an `e.` label to the issue (not the PR) to indicate 
-      the estimated effort required to fix the issue.
-  * If not green,
-    * Delete the merge commit, if any.
-    * Change the pull request status to `s.Ongoing`
-    * Add a comment to mention the test failure.
```

Comment:
The moment it is passed to the merger, there is no more "not green": only green (ready to merge), or grey due to conflicted or outdated branch, and setting it back to `s.Ongoing` because of such is hardly worth the extra review process needed.